File: Iniciante/1113 - Crescente e Decrescente/Python/1113.py
def main():
    controle = 0
    while controle == 0:
        X, Y = input().split()
        X = int(X)
        Y = int(Y)
        if X == Y:
            controle = 1
        elif X > Y:
            print("Decrescente")
        elif Y > X:
            print("Crescente")

main()

File: Iniciante/1113 - Crescente e Decrescente/JavaScript/1113.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split("\n");

  let controle = 0;
  for (let i = 0; controle === 0; i++) {
    let entradaAtual = lines[i].split(" ");
    let X = parseInt(entradaAtual[0]);
    let Y = parseInt(entradaAtual[1]);

    if (X == Y) {
      controle = 1;
    } else if (X > Y) {
      console.log("Decrescente");
    } else if (Y > X) {
      console.log("Crescente");
    }
  }
}

main();
